views/accessibility-statement.html

```diff
@@ -0,0 +1,112 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Accessibility statement – {{ SERVICE_NAME }} - GOV.UK
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <h1 class="govuk-heading-l">
+                Accessibility statement for the 'Register an overseas entity and tell us about its beneficial owners' service
+            </h1>
+
+            <ul class="govuk-list govuk-list--bullet">
+                <li><a href="#about-service">About this service</a></li>
+                <li><a href="#how-accessible">How accessible this service is</a></li>
+                <li><a href="#reporting">Reporting accessibility problems with this service</a></li>
+                <li><a href="#enforcement">Enforcement procedure</a></li>
+                <li><a href="#technical-information">Technical information about this service’s accessibility</a></li>
+                <li><a href="#tested">How we tested this service</a></li>
+                <li><a href="#were-doing">What we’re doing to improve accessibility</a></li>
+            </ul>
+
+            <h2 class="govuk-heading-m" id="about-service">About this service</h2>
+
+            <p>This accessibility statement applies to the 'Register an overseas entity and tell us about its beneficial owners' service.</p>
```

Comment:
I think the <p> tags will need to have the govuk-body styling to keep the font the same as the other pages
<p class="govuk-body">

Reply:
Yes this will need to be done for all the <p> tags by the looks of it

Reply:
Yep, will do